src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -472,12 +473,12 @@ const useGroupInfo = () => {
     onReportGroup,
     setUsername,
     setSelectedUser,
-    openChatMessage,
     generateSystemChat,
     setNewParticipant,
     isAnonymousModalOpen,
     setIsAnonymousModalOpen,
     blockModalRef,
+    openChatMessage,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The order of exported properties has been changed. This should not affect functionality if object destructuring is used correctly when importing these properties. However, it's worth checking to make sure there are no issues caused by this reordering.


<!-- This is an auto-generated comment by OSS CodeRabbit -->